core/src/main/scripts/importer/validateData.py

```diff
@@ -3502,7 +3502,7 @@ class ResourceDefinitionValidator(Validator):
     # 'RESOURCE_ID', 'RESOURCE_TYPE', 'DISPLAY_NAME' are required
     REQUIRE_COLUMN_ORDER = False
     REQUIRED_HEADERS = ['RESOURCE_ID', 'RESOURCE_TYPE', 'DISPLAY_NAME']
-    NULL_VALUES = ["[not applicable]", "[not available]", "[pending]", "[discrepancy]", "[completed]", "[null]", "", "na"]
+    NULL_VALUES = ["[not applicable]", "[not available]", "[null]", "", "na"]
```

Comment:
@dippindots i imagine `pending/discrepancy/completed` were prolly there for a reason. Did someone confirm they were no longer necessary? Otherwise I would leave them. I mostly meant that when writing the resource importer/validator code u prolly don't want to include them, since they are kind of weird NULL_VALUES. Like if I were to load data I would be surprised that these type of values are considered NULL. For backwards compatibility you would like to leave them but when introducing new code I would leave them out. 

So maybe you have something like `NULL_VALUES` that make sense for all kinds of data and `OTHER_CLINICAL_ATTRIBUTE_NULL_VALUES` which are in addition to the NULL_VALUES and specific to clinical attributes only

Anyway this is not super important but just wanted to clarify. You can have a single NULL_VALUES list but then to be safe I would include the previous values `"[pending]", "[discrepancy]", "[completed]"`

Reply:
Hi @inodb, I don't think I totally understand what you mean. I am not sure whether we want users to have NULL_VALUES in their data file?

Reply:
Do you mean that we can remove the current NULL_VALUES definition?